### PR TITLE
SC-1431 - Added a finally clause for closing the response body...

### PR DESCRIPTION
**Pull request by mickael-caro-sonarsource**
...in case of an error.

### CODE REVIEW

File: api/src/main/java/org/sonarsource/scanner/api/internal/ServerConnection.java [hunk 1]
```diff
@@ -82,6 +82,8 @@ public void downloadFile(String urlPath, Path toFile) throws IOException {
     } catch (IOException | RuntimeException e) {
       Utils.deleteQuietly(toFile);
       throw e;
+    }finally {
```
Comment (dbmeneses) [inline api/src/main/java/org/sonarsource/scanner/api/internal/ServerConnection.java]: A space is missing

Comment (henryju) [reply api/src/main/java/org/sonarsource/scanner/api/internal/ServerConnection.java]: ResponseBody is Closeable, so this is maybe better to use a try-with-resources pattern?

Comment (mickael-caro-sonarsource) [reply api/src/main/java/org/sonarsource/scanner/api/internal/ServerConnection.java]: Ok i included the ResponseBody object in the try, let me know if it's ok.

Comment (mickael-caro-sonarsource) [reply api/src/main/java/org/sonarsource/scanner/api/internal/ServerConnection.java]: I have a doubt seeing documentations, there's also AutoCloseable class for that, i don't know if Closeable is enough for this kind of pattern

**Pull request merged**